CoinDock-api/app/Http/Controllers/V1/UserController.php

```diff
@@ -74,11 +74,15 @@ public function primaryCurrency(User $user)
   }
   public function topPerformer(User $user)
   {
-    if ($user->wallets->isEmpty()) {
+    if($user->wallets->count() == 1){
+      return Null;
```

Comment:
returning Null? 

it doesn't mean response for an end-point. We follow some formats to handle responses, please use them accordingly.

Reply:
done